.gitignore

```diff
@@ -1,8 +1,33 @@
+
+# Created by https://www.gitignore.io/api/java,gradle
+
+### Java ###
+*.class
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
```

Comment:
These are not necessary. If they are removed, we dont need the "!gradle-wrapper.jar" below.